manifests/l/Lapce/Lapce/0.0.12/Lapce.Lapce.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Lapce.Lapce
+PackageVersion: 0.0.12
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VC++2015-2022Redist-x64
+ReleaseDate: 2022-03-24
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/lapce/lapce/releases/download/v0.0.12/Lapce-windows.msi
+  InstallerSha256: D6A0208C531E0F75A3FBBD9939EB6EE30A6D6330D633E9B98F1A5DEEA48C0C0E
+  ProductCode: '{33BB89E1-F794-4048-8381-3237B7395092}'
```

Comment:
```suggestion
  ProductCode: '{13DB46F9-9626-41A4-A654-B6C61E9F0AF6}'
```

Reply:
The ProductCode from msi property differs from that from the registry, and both of them in each version differs from another. Maybe UpgradeCode is better in this scenario. 🤷

Reply:
When I look at the MSI and the Registry they do match, and they are what Esco posted
![image](https://user-images.githubusercontent.com/12611259/163837837-116872e9-d258-4f38-8c47-5a026408aa8d.png)